app/jobs/result_monitor_loader.rb

```diff
@@ -5,6 +5,10 @@ class ResultMonitorLoader
   def self.perform(pipeline_run_id, output)
     Rails.logger.info("Loading #{output} for pipeline run #{pipeline_run_id}")
     pr = PipelineRun.find(pipeline_run_id)
+
+    # Just return if version is not yet set (likely a run just starting)
+    return if pr.pipeline_version.blank?
```

Comment:
Why does this line not take care of it?
https://github.com/chanzuckerberg/idseq-web/blob/dbbc042c6a60958ee163f705262d01b3cb827c7b/app/models/pipeline_run.rb#L746

Reply:
Hmm yeah maybe it isn't the real issue. Let me replace this with a log line to monitor instead

Reply:
updated 👍